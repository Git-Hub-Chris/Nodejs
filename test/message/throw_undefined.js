'use strict';
var assert = require('assert');

throw undefined;
