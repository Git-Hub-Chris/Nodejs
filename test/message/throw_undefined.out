
*test*message*throw_undefined.js:3
throw undefined;
^
undefined
